data-prepper-plugins/translate-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/translate/TargetsParameterConfig.java

```diff
@@ -22,18 +23,28 @@ public class TargetsParameterConfig {
     private final Map<String, Object> individualMappings = new HashMap<>();
     private final Map<Pattern, Object> compiledPatterns = new HashMap<>();
     @JsonProperty("target")
+    @JsonPropertyDescription("The key that specifies the field in the output in which the translated value will be placed.")
     @NotNull
     @NotEmpty
     private String target;
     @JsonProperty("map")
+    @JsonPropertyDescription("A list of key-value pairs that define the translations. Each key represents a possible " +
```

Comment:
Can we use multi line strings?

Reply:
The descriptions are single sentences with no line breaks in them, I'm not sure multi line strings can help here.